app/views/permissions/show.html.erb

```diff
@@ -0,0 +1,43 @@
+<div class='container mt-5'>
+	<div class='mb-3'><b>Email : </b><%= @user.email%></div>
+	<div class='mb-3'><b>Username : </b><%= @user.username%></div>
+	<div class='mb-3'><b>Role : </b>
+		<% if @user.roles.first.present? %>
+			<%= @user.roles.last.name%><br>
+		<%else%>
+			no role assigned<br>
+		<%end%>
+	</div>
+	
+	<%= select_tag(:role, 
+	options_for_select(['admin','moderator','manager','viewer'],
```

Comment:
define these types of array ['admin','moderator','manager','viewer'] in a separate file. Create a file as constant and define the array/value. import the constant where ever u want

Reply:
completed